devices/Ssd13xx/samples/Program.cs

```diff
@@ -1,287 +1,23 @@
-    {
-        using Image<L16> image = Image.Load<L16>(image_name);
-        ssd1306.DisplayImage(image);
-        Thread.Sleep(1000);
-    }
-}
-
-void DisplayClock(Ssd1306 ssd1306)
-{
-    Debug.WriteLine("Display clock");
-    var fontSize = 25;
-    var font = "DejaVu Sans";
-    var fontsys = SystemFonts.CreateFont(font, fontSize, FontStyle.Italic);
-    var y = 0;
-
-    foreach (var i in Enumerable.Range(0, 100))
-    {
-        using (Image<Rgba32> image = new Image<Rgba32>(128, 32))
-        {
-            if (image.TryGetSinglePixelSpan(out SpanRgba32 imageSpan))
-            {
-                imageSpan.Fill(Color.Black);
-            }
-
-            image.Mutate(ctx => ctx
-                .DrawText(DateTime.UtcNow.ToString("HH:mm:ss"), fontsys, Color.White,
-                    new SixLabors.ImageSharp.PointF(0, y)));
-
-            using (Image<L16> image_t = image.CloneAs<L16>())
-            {
-                ssd1306.DisplayImage(image_t);
-            }
-
-            y++;
-            if (y >= image.Height)
-            {
-                y = 0;
-            }
-
-            Thread.Sleep(100);
-        }
-    }
-}
+using System.Device.I2c;
+using System.Diagnostics;
+using System.Threading;
 
-// Referencing https://stackoverflow.com/questions/6803073/get-local-ip-address
-string? GetIpAddress()
-{
-    // Get a list of all network interfaces (usually one per network card, dialup, and VPN connection).
-    NetworkInterface[] networkInterfaces = NetworkInterface.GetAllNetworkInterfaces();
 
-    foreach (NetworkInterface network in networkInterfaces)
-    {
-        // Read the IP configuration for each network
-        IPInterfaceProperties properties = network.GetIPProperties();
 
-        if (network.NetworkInterfaceType == NetworkInterfaceType.Ethernet &&
-            network.OperationalStatus == OperationalStatus.Up &&
-            !network.Description.ToLower().Contains("virtual") &&
-            !network.Description.ToLower().Contains("pseudo"))
-        {
-            // Each network interface may have multiple IP addresses.
-            foreach (IPAddressInformation address in properties.UnicastAddresses)
-            {
-                // We're only interested in IPv4 addresses for now.
-                if (address.Address.AddressFamily != AddressFamily.InterNetwork)
-                {
-                    continue;
-                }
 
-                // Ignore loopback addresses (e.g., 127.0.0.1).
-                if (IPAddress.IsLoopback(address.Address))
-                {
-                    continue;
-                }
+Debug.WriteLine("Hello Ssd1306 Sample!");
 
-                return address.Address.ToString();
-            }
-        }
-    }
 
-    return null;
-}
+//Tested with 128x64 and 128x32 OLEDs
+using Ssd1306 device = new Ssd1306(I2cDevice.Create(new I2cConnectionSettings(2, 0x3C)), Ssd13xx.DisplayResolution.OLED128x64);
```

Comment:
For the I2C address, a good pattern is to add in the Ssd1306 class something like: public const int DefaultI2cAddress = 0x3C;

Reply:
Also as this is using I2C, please look at any other sample using I2C, there is a special commented part to add at the beginning of the code to help people understand that they need to setup properly the pins if needed. See for example: https://github.com/nanoframework/nanoFramework.IoT.Device/blob/04ac94cefb39f70312ada24b12ae7cb289cbe219/devices/Bmxx80/samples/Bmp280.sample.cs#L26